ember-flight-icons/tests/dummy/app/templates/application.hbs

```diff
@@ -35,12 +33,15 @@
     <ul class="ds-ul-grid">
       {{#each @model as |meta|}}
         <li class="ds-li {{if meta.isHidden 'd-none'}}">
-          <FlightIcon
-            @name={{Onlyname meta.name}}
-            @size={{if (eq meta.size 16) '16'}}
-            class="demo-icon"
-          />
-          {{meta.name}}
+          <div class="icon-frame">
+            <FlightIcon
+              @name={{Onlyname meta.name}}
+              @size={{if (eq meta.size '16') '16'}}
+              @color="currentColor"
```

Comment:
@MelSumner I think something in the icon component logic might be off - I couldn't inherit colours unless I explicitly passed `currentColor` to the `@color` property which is why I added this line.

Reply:
Looking at [this line](https://github.com/hashicorp/flight/blob/c0ac030f259bdf11c4c761a76fa83c584a73e8ee/ember-flight-icons/addon/components/flight-icon.js#L19) in the component logic it seems like it should be falling back to `currentColor` but I wasn't seeing that in the dummy app when using the component - it didn't seem to output a `fill` property on the SVG DOM element unless explicitly defined.

Reply:
oh I see what it is... in `flight-icon.hbs` I think it should be `fill={{this.color}}` - right now, the template says "only give me what is passed to me". I'll fix that in a separate PR. 👍

Reply:
ok: https://github.com/hashicorp/flight/pull/103

Reply:
```suggestion
```
Can you try removing this line, you shouldn't need it after the bug fix. 👍